test/fail_compilation/test17908.d

```diff
@@ -0,0 +1,15 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/test17908.d(10): Error: function test17908.foo is not callable because it is annotated with @disable
+---
+*/
+
+@disable void foo();
+@disable void foo(int) {}
+alias g = foo;
+
+void main()
+{
+    g(10);
+}
```

Comment:
Could you add a test with one member of the overload set being disabled (out of two), and calling the disabled overload ?

Reply:
Done, thanks for the tip